src/services/dexes/uniswapV3/UniswapV3LiquidityResolver.ts

```diff
@@ -16,6 +15,15 @@ import {UniswapV3Pool} from '../../../models/UniswapV3Pool.js';
 import {FetcherName} from '../../../types/fetchers.js';
 import {DeviationLeavesAndFeeds} from 'src/types/DeviationFeeds.js';
 
+export const UniswapV3ChainNumber: Partial<Record<ChainsIds, number>> = {
```

Comment:
why we need this?

Reply:
This is outdated, it is on `getUniswapV3ChainNumber`.
We need to get the chainNumber to create Token istance